recipes/libepoxy/all/conanfile.py

```diff
@@ -90,6 +94,11 @@ def _configure_meson(self):
             defs[opt] = "true" if self.options.get_safe(opt, False) else "false"
         args=[]
         args.append("--wrap-mode=nofallback")
+
+        if self._is_msvc:
+            # Fix error C4819
+            args.append("-Dc_args=/source-charset:utf-8")
+
```

Comment:
it seems to be fixed in master branch: https://github.com/anholt/libepoxy/blob/master/meson.build#L98

So maybe put this in a patch of meson.build instead?

Reply:
That sounds a good idea.